core/tests/test_utils.py

```diff
@@ -197,11 +235,18 @@ def _restore_stashed_user_env(self):
         self.stashed_user_env = None  # pylint: disable=attribute-defined-outside-init
 
     def login(self, email, is_super_admin=False):
+        """Sets the environment variables to simulate a login.
+
+        Args:
+            email: str. The email of the user who is to be logged in.
+            is_super_admin: bool. Whether the user is a super admin.
+        """
         os.environ['USER_EMAIL'] = email
         os.environ['USER_ID'] = self.get_user_id_from_email(email)
         os.environ['USER_IS_ADMIN'] = '1' if is_super_admin else '0'
 
     def logout(self):
+        """Resets USER_EMAIL, USER_ID and USER_IS_ADMIN."""
```

Comment:
This is an implementation detail. Maybe say: "Simulates a logout by resetting the environment variables."